### PR TITLE
Group study

### PR DESCRIPTION
**Pull request by AshwiniDhaneppagol**
Group file is ready to review

### CODE REVIEW

File: src/Room3/GroupTask/File1.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package Room3.GroupTask;
+
+public class File1 {
+    public static void main(String[] args) {
+        int a=2;
```
Comment (yavuzvelioglu) [inline src/Room3/GroupTask/File1.java]: Hi @AshwiniDhaneppagol please 2 is noy my fav number change to 7

Comment (yavuzvelioglu) [reply src/Room3/GroupTask/File1.java]: Ok @AshwiniDhaneppagol it has done

**Review by yavuzvelioglu — APPROVED** [review]
Ok @AshwiniDhaneppagol it has done well done

File: src/Room3/GroupTask/File1.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package Room3.GroupTask;
+
+public class File1 {
+    public static void main(String[] args) {
+        int a=7;
+        int b=4;
+        System.out.println(a+b);
```
Comment (emrah-ilgar) [inline src/Room3/GroupTask/File1.java]: can you change "+" operator with "*" operator?

**Review by emrah-ilgar — APPROVED** [review]
it is goog for me thanks @AshwiniDhaneppagol

File: src/Room3/GroupTask/File1.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package Room3.GroupTask;
+
+public class File1 {
+    public static void main(String[] args) {
+        int a=7;
+        int b=4;
```
Comment (MonikaSturman) [inline src/Room3/GroupTask/File1.java:6]: @AshwiniDhaneppagol please change the number from 7 to 1 please

Comment (AshwiniDhaneppagol) [reply src/Room3/GroupTask/File1.java:6]: Updated file code --> 2a1333d07e95ee52f45a1a52cb5d58b52d996089

Comment (MonikaSturman) [reply src/Room3/GroupTask/File1.java:6]: @AshwiniDhaneppagol all done thank you

**Review by MonikaSturman — APPROVED** [review]

**Pull request merged**